examples/composite-alarm/main.tf

```diff
@@ -0,0 +1,110 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+locals {
+  tags = {
+    example = "composite-alarm"
+    region  = "eu-west-1"
+  }
+}
+
+module "composite_alarm" {
+  source = "../../modules/composite-alarm"
+
+  alarm_name        = "composite-alarm"
+  alarm_description = "Example of a composite alarm"
+
+  alarm_actions = [module.aws_sns_topic_2.sns_topic_arn]
+  ok_actions    = [module.aws_sns_topic_2.sns_topic_arn]
+
+  # https://github.com/hashicorp/terraform-provider-aws/issues/19575
```

Comment:
```suggestion
```
Our module is not affected by this issue reported in the provider. It affects their documentation.